mbed-util/FunctionPointerBase.h

```diff
@@ -38,6 +46,7 @@ class FunctionPointerBase {
     virtual void clear() {
         _membercaller = NULL;
         _object = NULL;
+        memset(_member, 0, sizeof(_member));
```

Comment:
Given that _member is guaranteed to be 4-byte aligned, is memset the best choice here?